### ISSUE TITLE
Problems compiling SDL with KMSDRM on Raspberry Pi 4: EGL_PLATFORM_GBM_MESA undeclared

### ISSUE THREAD

**Issue by gepatto** [libsdl-org/SDL#4232]
I'm trying to compile SDL-master on a Raspberry Pi 4 with configuration  
`./configure --enable-video-kmsdrm` 

It fails with the following error:
```
SDL_kmsdrmvideo.c:1204:56: error: ‘EGL_PLATFORM_GBM_MESA’ undeclared (first use in this function); did you mean ‘VK_PLATFORM_H_’?
      if (SDL_EGL_LoadLibrary(_this, NULL, egl_display, EGL_PLATFORM_GBM_MESA)) {
                                                        ^~~~~~~~~~~~~~~~~~~~~
                                                        VK_PLATFORM_H_
```

I tried `make` from the rootdir  and also `cmake ../ && make`  from a build directory. 
Both give the same error

I see that EGL_PLATFORM_GBM_MESA is defined in SDL_kmsdrmopengles.c

Do I need another configure option or is this a bug?

When I copy 
```
#ifndef EGL_PLATFORM_GBM_MESA
#define EGL_PLATFORM_GBM_MESA 0x31D7
#endif
```
over to SDL_kmsdrmvideo.c, It compiles without errors and I can use SDL

**Comment by sezero** [comment]
Looks like this is a problem with 2.0.14 release: current git doesn't have the issue.
As for copying the missing define, looks like a fair way of fixing it within 2.0.14.

**Comment by sezero** [comment]
Closing this one.

**Comment by gepatto** [comment]
I have this problem even with todays commit.  not 2.0.14

**Comment by sezero** [comment]
My mistake. Re-opening.

**Comment by sezero** [comment]
Should be fixed by commit 5262b52ddd83b88544ebff747cf33e9a295cfc0a

**Comment by gepatto** [comment]
Yes, fixed.. Thanks!
